src/FreeFormIdeas.css

```diff
@@ -0,0 +1,11 @@
+.FreeFormIdeas {
+  top: inherit;
```

Comment:
Do we need this?

Reply:
Not anymore now that you helped me with `getBoundingClientRectangle()`